src/Resources/config/services.yaml

```diff
@@ -2,4 +2,4 @@ services:
     Bref\Symfony\Messenger\Service\BusDriver: '@Bref\Symfony\Messenger\Service\SimpleBusDriver'
     Bref\Symfony\Messenger\Service\SimpleBusDriver:
         arguments:
-            - '@logger'
+            - '@?logger'
```

Comment:
Why do we make it optional?

Reply:
This service might not exist. That is why I made this optional.

Reply:
Oh ok I thought that service always existed in Symfony.